appveyor.yml

```diff
@@ -0,0 +1,15 @@
+version: '{build}'
+build_script:
+- cmd: >-
+    set RUBY_VERSION=200
+
+    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+
+    gem install bundler
+
+    bundle install
+
+    build.netstd.cmd
+test: off
+on_finish:
+    - ps: $root = Resolve-Path .\artifacts; [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName to-publish }

```

Comment:
The `on_finish` ensures that the entire artifacts directory is published even on a failed build. Without this, we get no logs for failed builds.

I can do the install/init split and check the environment vars. I don't have an opinion on where `gem install bundler` should go, but will do learning and try to make one.
I believe things didn't work well without the `gem install bundler`. In fact, I think there was a message in the build telling me to run it before I had that line, but I will try again without it to see what happens.